### PR TITLE
added openmaptiles + osmnames

### PR DESCRIPTION
**Pull request by xkomczax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -520,13 +520,15 @@ Table of Contents
...
-   * [datamaps.co](https://datamaps.co/) a free platform for creating visualizations with data maps
+   * [datamaps.co](https://datamaps.co/) — a free platform for creating visualizations with data maps
    * [geocod.io](http://geocod.io/) — Geocoding via API or CSV Upload. 2,500 free queries/day
    * [gogeo.io](http://gogeo.io/) — Maps and geospatial services with an easy to use API and support for big data
    * [carto.com](https://carto.com/) — Create maps and geospatial APIs from your data and public data
    * [giscloud.com](http://giscloud.com/) — Visualize, analyze and share geo data online
    * [latlon.io](https://latlon.io/) — Geocoding API + school districts, census geography divisons and other address based data. 2,500 free requests/month
    * [mapbox.com](https://mapbox.com/) — Maps, geospatial services and SDKs for displaying map data
+   * [openmaptiles](https://openmaptiles.org/) - Set of open-source tools for self-hosting of OpenStreetMap maps in more than 50 languages. It provides both raster as well as vector tiles, WMS and WMTS services for GIS programs, support for JavaScript viewers and mobile SDK
```
Comment (ripienaar) [inline README.md]: not really any self hosting software here so maybe remove this one? the rest are good

**Comment by xkomczax** [comment]
removed openmaptiles, added tilehosting.com

File: README.md [hunk 2]
```diff
@@ -520,13 +520,15 @@ Table of Contents
...
-   * [datamaps.co](https://datamaps.co/) a free platform for creating visualizations with data maps
+   * [datamaps.co](https://datamaps.co/) — a free platform for creating visualizations with data maps
    * [geocod.io](http://geocod.io/) — Geocoding via API or CSV Upload. 2,500 free queries/day
    * [gogeo.io](http://gogeo.io/) — Maps and geospatial services with an easy to use API and support for big data
    * [carto.com](https://carto.com/) — Create maps and geospatial APIs from your data and public data
    * [giscloud.com](http://giscloud.com/) — Visualize, analyze and share geo data online
    * [latlon.io](https://latlon.io/) — Geocoding API + school districts, census geography divisons and other address based data. 2,500 free requests/month
    * [mapbox.com](https://mapbox.com/) — Maps, geospatial services and SDKs for displaying map data
+   * [osmnames](https://osmnames.org/) - Geocoding, search results ranked by the popularity of related Wikipedia page
+   * [tilehosting](https://www.tilehosting.com/) - Vector maps, map services and SDKs for map visualisation
```
Comment (ripienaar) [inline README.md]: great, can you add some info about what this free plan allow?

**Comment by xkomczax** [comment]
you can read more about the plans here: https://www.tilehosting.com/#plans 

summary: it gives you vector tiles, regular updates, and four default styles. Other map services (like WMTS), custom raster styles, and premium datasets (hillshading, satellite imagery, ...) are not provided.

**Comment by ripienaar** [comment]
Yeah I saw those but I mean in the entry. You’ll see ms y others have quick blurb on what you get free

**Comment by xkomczax** [comment]
Ah, yes, sure! Done...

**Comment by ripienaar** [comment]
thank you!

**Pull request merged**